tests/compile/buildenv.pyx

```diff
@@ -2,8 +2,6 @@
 Non-test that prints debug information about the current build environment.
 """
 
-from __future__ import print_function
```

Comment:
Not sure we can remove this - most of the tests are run in `language_level=2` so this is overriding a specific bit of that.

The same applies for the other uses of `print_function`. You'll probably find out if it's important when the CI runs though